src/components/Banner/Banner.less

```diff
@@ -28,17 +28,12 @@
 		border-radius: 0;
 	}
 
```

Comment:
I didn't have good idea if there's a preferred convention for overriding prior-set CSS styles, so we went with "beefing" up this selector so it overrode the prior

Reply:
Although this works, generally we try to add a new class to the existing component and reference it that way. In other words, you should be able to replace `.lucid-Icon-is-clickable` with `.lucid-Banner-close` since you've already added that class.

This prevents the case where someone putting a different icon inside their own banner might have their styles stomped on.